Dockerfile

```diff
@@ -1,18 +1,22 @@
-FROM maven:3-jdk-8-alpine
+FROM maven:3-jdk-8
 LABEL maintainer="BaseX Team <basex-talk@mailman.uni-konstanz.de>"
 
 # Compile BaseX, install
 COPY . /usr/src/basex/
 
 # install git as "buildnumber-maven-plugin" requires git:
-RUN apk update && apk add --no-cache git && \
+RUN apt-get update && \
+    apt-get install -y git && \
+    apt-get -y autoremove && \
+    rm -r /var/lib/apt/lists/* /var/cache/* && \
     cd /usr/src/basex && \
     mvn clean install -DskipTests && \
     ln -s /usr/src/basex/basex-*/etc/* /usr/local/bin &&\
-    adduser -h /srv -D -u 1984 basex && \
+    yes "" | adduser --home /srv --disabled-password --disabled-login --uid 1984 basex && \
```

Comment:
Since `adduser` is interactive, it's not required but we could consider using `useradd` here instead. That would change
```
    yes "" | adduser --home /srv --disabled-password --disabled-login --uid 1984 basex
```
to
```
groupadd --gid 1984 basex && useradd --system --uid 1984 --gid 1984 --home-dir /srv basex
```
(The groupadd is required to get the exact gid, I think, and there is an obscure reason for this exact gid.)